Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/textfile/TextFileDAO.java

```diff
@@ -0,0 +1,63 @@
+package edu.asu.spring.quadriga.dao.impl.textfile;
+
+import org.hibernate.SessionFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import edu.asu.spring.quadriga.dao.impl.BaseDAO;
+import edu.asu.spring.quadriga.dao.impl.UserDAO;
+import edu.asu.spring.quadriga.dao.textfile.ITextFileDAO;
+import edu.asu.spring.quadriga.dto.TextFileDTO;
+import edu.asu.spring.quadriga.dto.WorkspaceDTO;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+
+@Repository
+public class TextFileDAO extends BaseDAO<TextFileDTO> implements ITextFileDAO {
+
+    @Autowired
+    private SessionFactory sessionFactory;
+
+    private static final Logger logger = LoggerFactory.getLogger(TextFileDAO.class);
+
+    @Override
+    public TextFileDTO getTextFileDTO(String wsId) {
+        // TODO Auto-generated method stub
+        return null;

```

Comment:
Defined these methods to work on these in the next story to retrieve/ display text files.